### DISCUSSION TITLE
Windows Server Core with Desktop Experience

### DISCUSSION THREAD

**Discussion by baendron** [microsoft/dotnet-framework-docker#1016] [Q&A]
Good day, 

I am currently trying to convert an existing ASP.NET framework project to use Docker. In the process of doing so, I've run into an issue where my project requires Desktop Experience to run properly. I have been trying to search for base images for Windows Server Core with this feature enabled but I am struggling to find anything related to this. 

I am currently using `mcr.microsoft.com/windows/servercore:ltsc2019` but this does not seem to have Desktop Experience enabled. Is anyone able to provide any help on how to enable Desktop Experience with this image or provide a link to an image that has that feature enabled?

**Comment by mthalman** [comment]
See [Server Core vs Server with Desktop Experience install options](https://learn.microsoft.com/windows-server/get-started/install-options-server-core-desktop-experience) for the differences between Server Core and Server with Desktop Experience.

We only provide .NET Framework images for Server Core. Windows does offer [Server container images](https://hub.docker.com/_/microsoft-windows-server) starting with Server 2022. But I don't know how Desktop Experience manifests within container images, if at all. I guess it may depend on what your actual dependencies are. You may want to follow up with the [Windows Containers](https://github.com/Microsoft/Windows-Containers) team to get more detail on that.
